site/content/blogs/release-v132.en.md

```diff
@@ -18,7 +18,7 @@ Thanks to every one of you who shows love and support for AWS Copilot.
 
 Copilot v1.32 brings big enhancements to help you develop more flexibly and efficiently:
 
-- **`copilot run local --proxy`**:
+- **`copilot run local --proxy`**: Proxy outbound traffic from your local containers to services and RDS instances in your environment VPC. [See detailed section](#copilot-run-local---proxy)
```

Comment:
Should we leave VPC out and use Copilot environment instead? As i would assume the traffic is still controlled by the env sg which means two env cannot talk to each other even if they share the same VPC by default.